src/index.test.ts

```diff
@@ -125,19 +128,26 @@ test('Response data can be deduplicated with graphql-deduplicator', async t => {
   }).promise()
 
   const deduplicated = await request({
+    uri: uri + '?deduplicate',
+    method: 'POST',
+    json: true,
+    body: { query },
+  }).promise()
+
+  const deduplicatedWithHeader = await request({
```

Comment:
It might be useful to state why the test is here. Others might be surprised seeing this despite that the docs don't mention `X-GraphQL-Deduplicate`.

Reply:
Right, a comment can make it more clear